### ISSUE TITLE
When importing movie ratings from IMDB, 2 movies keep stacking plays

### ISSUE THREAD

**Issue by DeshMindER** [damienhaynes/TraktRater#34]
Hi.

Everytime I run the import tool using the .csv from IMDB movie ratings, 2 specific movies keep adding playtimes, instead of just ignoring, like the others on the .csv file.

Down below is the log with the 2 movies.

2015-05-26 20:58:43.696 [INFO] [][01]: Found 1391 watched movies on trakt
2015-05-26 20:58:43.697 [INFO] [][01]: Filtering out watched movies that are already watched on trakt.tv
2015-05-26 20:58:43.999 [INFO] [][01]: Importing 2 IMDb movies as watched...
2015-05-26 20:58:44.000 [INFO] [][01]: Importing page 1/1 IMDb movies as watched...
2015-05-26 20:58:44.000 [DEBG] [][05]: Address: http://api.trakt.tv/sync/history, Post: {"movies":[{"ids":{"slug":null,"trakt":null,"imdb":"tt2332518","tmdb":null},"title":"Até que a Sorte nos Separe","year":2012,"watched_at":"2013-01-28T00:00:00Z"},{"ids":{"slug":null,"trakt":null,"imdb":"tt1568829","tmdb":null},"title":"Herbert de Perto","year":2009,"watched_at":"2011-11-10T00:00:00Z"}]}
2015-05-26 20:58:44.465 [DEBG] [][05]: Response: {"added":{"movies":2,"episodes":0},"not_found":{"movies":[],"shows":[],"seasons":[],"episodes":[],"people":[]}}
2015-05-26 20:58:44.466 [INFO] [][01]: Finished import from IMDb
2015-05-26 20:58:44.488 [INFO] [][01]: Import Complete!

So if I try to run the tool 3 times in a row, it will add 3 plays foe each movie above, for some unknown reason.

**Comment by damienhaynes** [comment]
Having a look at that POST, I can see that TraktRater is doing the correct thing. If you have a look at the movie "Até que a Sorte nos Separe" on trakt: http://trakt.tv/movies/ate-que-a-sorte-nos-separe-2012, you will notice that the IMDb ID is missing.

Having a look at the corresponding movie on themoviedb.org the IMDb ID is correct. I have triggered a refresh for that movie since im a VIP member.

The movie "Herbert de Perto" is also missing an IMDb ID on trakt: http://trakt.tv/movies/herbert-de-perto-1969. Having a look at the corresponding entry on themoviedb.org it is also missing an IMDb ID. I have edited the entry on themoviedb.org "https://www.themoviedb.org/movie/58209-herbert-de-perto" to have both a valid IMDb ID and a Year. This movie will need a refresh on trakt as well but might be a few days until done.
